### PR TITLE
E summit Page -> UI

### LINKED ISSUE #62: E-Summit Screen -> UI and Linking
### Is your feature request related to a problem? Please describe.

Creating the UI of the screen and then linking the completed screen to the correct position in the widget tree of the app.

### PR DESCRIPTION
**Pull request by Hard-Coder05**
### Description

**Type of Change:** (Delete irrelevant options)
- Code
- User Interface

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

on local device

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
Screenshot_2021-02-09-21-20-26-566_com nitrr ecell esummit ecellapp 1

### CODE REVIEW

**Comment by werainkhatri** [comment]
https://gist.github.com/werainkhatri/511c9c0618c5fd06be75d52e9a9d676b

**Comment by Hard-Coder05** [comment]
@werainkhatri Sir I have done all necessary changes in the UI. This screen looks good to me.

**Comment by Hard-Coder05** [comment]
> @werainkhatri Sir I have done all necessary changes in the UI. This screen looks good to me.
> 
> !(width="250" src="https://user-images.githubusercontent.com/54059881/107465515-840b7600-6b88-11eb-8d08-d2ceaff5f260.jpg")

File: lib/screens/esummit/esummit.dart [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                                    style: TextStyle(
+                                        fontSize: 20 * heightFactor,
+                                        fontWeight: FontWeight.w300,
+                                        height: 1.2,
+                                        wordSpacing: 4),
+                                    children: <TextSpan>[
+                                  TextSpan(
+                                      text: "E-Summit ",
+                                      style: TextStyle(
+                                          color: C.speakerButtonColor,
```
Comment (werainkhatri) [inline lib/screens/esummit/esummit.dart]: please coordinate with mohit about colors. I had a talk and he told me color here is different from the button color. `#5D02A3`

File: lib/screens/esummit/esummit.dart [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                                                      color: C.primaryUnHighlightedColor,
+                                                      fontSize: 20 * heightFactor,
+                                                      fontWeight: FontWeight.w300)),
+                                              SizedBox(width: heightFactor * 10),
+                                              Icon(Icons.arrow_forward_ios,
+                                                  color: C.primaryUnHighlightedColor,
+                                                  size: heightFactor * 20)
+                                            ])))))
+                      ]))
+                ]))));
```
Comment (werainkhatri) [inline lib/screens/esummit/esummit.dart]: add commas. Refer the gist i commented in the conversation.

File: lib/screens/esummit/esummit.dart [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            child: Container(
+                decoration: BoxDecoration(
+                    gradient: LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  stops: [0.3, 1],
+                  colors: [C.backgroundTop1, C.backgroundBottom1],
+                )),
+                child: Column(mainAxisSize: MainAxisSize.min, children: [
+                  SizedBox(height: height / 6),
```
Comment (werainkhatri) [inline lib/screens/esummit/esummit.dart]: there is no need for this. refer the gist how I have done it.

**Review by werainkhatri — CHANGES_REQUESTED** [review]
Refer the gist and implement some code styles I did.

File: lib/screens/esummit/esummit.dart (hunk 2 above)
Comment (Hard-Coder05) [reply lib/screens/esummit/esummit.dart]: done sir!

File: lib/screens/esummit/esummit.dart (hunk 1 above)
Comment (Hard-Coder05) [reply lib/screens/esummit/esummit.dart]: as asked from mohit he told to use #8E31D5 for both the speaker button and the highlighted text.

Comment (werainkhatri) [reply lib/screens/esummit/esummit.dart]: Coordinate with mohit. He''ll update you.

Comment (Hard-Coder05) [reply lib/screens/esummit/esummit.dart]: i had a talk with him just after your review he said to use the #8E31D5 color for both.

**Comment by Hard-Coder05** [comment]
Final UI from my side

**Comment by Hard-Coder05** [comment]
@werainkhatri done from my side!

**Review by werainkhatri — APPROVED** [review]

**Pull request merged**